.changeset/strong-penguins-kick.md

```diff
@@ -0,0 +1,21 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+Our CSS custom properties have been updated to follow a new semantic token format:
+
+```
+--glide-core-{collection}-{category*}-{scope*}-{property}-{variant*}--{state*}
```

Comment:
Worth concisely defining what each semantic part means or providing an example variable? 

I ask because I can imagine people reading this and immediately having questions or assuming that the terms map to technical terms.

Reply:
Great call.  I'll cook something up here in a bit and get your thoughts!

Reply:
Let me know what you think - this is directly from design: https://github.com/CrowdStrike/glide-core/pull/750/commits/714040bd379c9710c23eed17b7c03f157d67bd7c